pkg/ui/yarn.lock

```diff
@@ -3312,7 +3312,11 @@ typedarray@~0.0.5:
   version "0.0.6"
   resolved "https://registry.yarnpkg.com/typedarray/-/typedarray-0.0.6.tgz#867ac74e3864187b1d3d47d996a78ec5c8830777"
 
-typescript@^2.0.3, typescript@^2.0.9:
+typescript:
```

Comment:
There were previously two locks, for typescript ^2.0.3 and ^2.0.9.   The locks don't match up, i'm not quite sure which tools are using different versions of typescript; although apparently, they were already using 2.1.1.